bridge/qb/server/functions.lua

```diff
@@ -2,6 +2,9 @@ require 'server.functions'
 require 'bridge.qb.server.player'
 local functions = {}
 
+local allowMethodOverrides = GetConvarInt('qbx:allowmethodoverrides', 1) == 1
+local disableMethodOverrideWarning = GetConvarInt('qbx:disableoverridewarning', 0) == 1
```

Comment:
these should be `true` or `false`

Reply:
https://github.com/overextended/ox_inventory/blob/cf88913cfffe3122a6655ed4af04b18cf159ea11/init.lua#L79-L80

I don't think it works the other way with the `GetConvar()` as it would return a string of `'true'` vice `true`

Reply:
im saying change it to GetConvar and use boolean values like how we do it [here](https://github.com/Qbox-project/qbx_core/blob/main/server/player.lua#L7)